kotlin-codegen/integration-test/src/test/kotlin/com/squareup/moshi/kotshi/GenericClass.kt

```diff
@@ -0,0 +1,7 @@
+package com.squareup.moshi.kotshi
```

Comment:
I’m reluctant to merge this unless it’s a contribution from the original author. CLA stuff.

Reply:
@ansman ?